contents/blog/posthog-vs-heap.md

```diff
@@ -0,0 +1,834 @@
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Use analytics without cookie consent banners</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+- **HIPAA compliance:** Heap will sign a Business Associate Agreement if you require HIPAA compliance. At PostHog, we currently recommend self-hosting PostHog if you require [HIPAA compliance](/docs/privacy/hipaa-compliance). This ensures all sensitive data never leaves your infrastructure, giving you complete control and simplifying compliance processes. HIPAA certification for PostHog Cloud is planned in 2023.
+
+- **Cookie-less tracking:** If you'd rather not serve annoying cookie banners to your users, PostHog can be configured to store data in memory so it doesn't persist beyond a single session. See: [How to use PostHog without cookie banners](/tutorials/cookieless-tracking). Some forms of user analysis aren't possible with this method.
+
+### Admin & security
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+            <td className="w-3/12"></td>
+            <td><strong>Mixpanel</strong></td>
+            <td><strong>PostHog</strong></td>
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>SOC 2 certification</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Externally audited security</td>
+        </tr>
+        <tr>
+            <td><strong>SAML/SSO</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Organization single sign-on</td>
+        </tr>
+        <tr>
+            <td><strong>User permissions</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Assign access levels to team members</td>
+        </tr>
+        <tr>
+            <td><strong>Multiple organizations</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Create orgs for all your internal teams</td>
+        </tr>
+        <tr>
+            <td><strong>Multiple projects</strong></td>
+            <td className="text-center">2 (pay for more)</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Track multiple products</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+<ArrayCTA />
+
+## Frequently asked questions
```

Comment:
Can you use both for ecommerce sites? Or some question about ecommerce sites.

Reply:
What do their data governance policies look like?

Reply:
How scalable is Heap and PostHog?

Reply:
Not sure we want to into the weeds of data governance, but I'll add the other two.

Reply:
Although Heap has a section on their site for "data governance" it doesn't seem more complicated than our project settings page 😅